File: manualPackage/mappls-map-react-native/android/rctmgl/src/main/java/com/mappls/sdk/maps/rctmgl/components/geoAnalytics/StyleRequest.java
package com.mappls.sdk.maps.rctmgl.components.geoAnalytics;

public class StyleRequest {
      private String labelColor;
      private Integer labelSize;
      private String fillColor;
      private Integer pointSize;
      private String  strokeColor;
      private Double strokeWidth;
      private Double fillOpacity;

    public String getLabelColor() {
        return labelColor;
    }

    public void setLabelColor(String labelColor) {
        this.labelColor = labelColor;
    }

    public Integer getLabelSize() {
        return labelSize;
    }

    public void setLabelSize(Integer labelSize) {
        this.labelSize = labelSize;
    }

    public String getFillColor() {
        return fillColor;
    }

    public void setFillColor(String fillColor) {
        this.fillColor = fillColor;
    }

    public Integer getPointSize() {
        return pointSize;
    }

    public void setPointSize(Integer pointSize) {
        this.pointSize = pointSize;
    }

    public String getStrokeColor() {
        return strokeColor;
    }

    public void setStrokeColor(String strokeColor) {
        this.strokeColor = strokeColor;
    }

    public Double getStrokeWidth() {
        return strokeWidth;
    }

    public void setStrokeWidth(Double strokeWidth) {
        this.strokeWidth = strokeWidth;
    }

    public Double getFillOpacity() {
        return fillOpacity;
    }

    public void setFillOpacity(Double fillOpacity) {
        this.fillOpacity = fillOpacity;
    }
}

File: manualPackage/mappls-map-react-native/ios/RCTMGL/StyleRequest.m
//
//  StyleRequest.m
//  mappls-map-react-native
//
//  Created by Shashank on 3/3/22.
//

#import "StyleRequest.h"

@implementation StyleRequest


-(void)setLabelColor:(NSString *)labelColor {
    _labelColor = labelColor;
}

-(void)setLabelSize:(NSNumber *)labelSize {
    _labelSize = labelSize;
}

-(void)setFillColor:(NSString *)fillColor {
    _fillColor = fillColor;
}

-(void)setPointSize:(NSNumber *)pointSize {
    _pointSize = pointSize;
}

-(void)setStrokeColor:(NSString *)strokeColor {
    _strokeColor = strokeColor;
}

-(void)setStrokeWidth:(NSNumber *)strokeWidth {
    _strokeWidth = strokeWidth;
}

-(void)setFillOpacity:(NSNumber *)fillOpacity {
    _fillOpacity = fillOpacity;
}


@end
